### DISCUSSION TITLE
nobara linux

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [end-4/dots-hyprland#539] [Extra Distros]
how do i use this on [nobara linux](https://nobaraproject.org/)

**Comment by AKA-Akhil** [comment]
it should be the same for almost every arch based distro i think,just go through the [wiki](https://end-4.github.io/dots-hyprland-wiki/en/i-i/01setup/)

**Reply by pizzav-xyz** [reply]
Nobara is fedora based

**Reply by pizzav-xyz** [reply]
hello @AKA-Akhil ?

**Comment by pizzav-xyz** [comment]
Fedora based

**Comment by end-4** [comment]
No guarantees, but try this
https://end-4.github.io/dots-hyprland-wiki/en/i-i/01setup/#manual-installation-basically-any-distro
